arithmaticOperation.js

```diff
@@ -6,7 +6,9 @@
 *   
 *	Return a number denoting the rectangle's area.
 **/
-function getArea(length, width) {
+function getArea(length, width){
+    if(length===undefined || width===undefined || isNaN(length) || isNaN(width))
```

Comment:
why you are using isNaN and undefined at the same time ?

isNaN(43)  => return false
isNaN(undefined)  =>  return true

Modify your logic

Reply:
ok I will do that

Reply:
It is working fine for me

Reply:
No. it is not working fine . 

if you give input to the function like this , you won't get correct output :   getArea(``,``)